scripts/package.json

```diff
@@ -47,7 +47,6 @@
     "@types/puppeteer": "1.12.3",
     "@types/read-pkg-up": "^6.0.0",
     "async": "^2.6.1",
-    "autoprefixer": "^10.2.1",
```

Comment:
Why the downgrade from autoprefixer 10 to 9? (looking at the root package.json)

Reply:
Due to changes in 6.2 that required us to manually add `autoprefixer` as a dependency, I wanted to add the `autoprefixer` version that storybook used to automatically include which is v9. This was also an issue tracked [here](https://github.com/storybookjs/storybook/issues/12668) that was supposed to be solved by simply installing `addons-postcss`  but clearly was not in this case.

![image](https://user-images.githubusercontent.com/8649804/124197991-97aa7100-da84-11eb-977b-549a8e4fb06f.png)

Was also getting this error below when I used `autoprefixer: 10.2.1`:

![image](https://user-images.githubusercontent.com/8649804/124197729-089d5900-da84-11eb-93e9-ab72de2215d1.png)

Reply:
`autoprefixer` 10 in scripts was the version that was actually being used on our production code, so at least in my opinion it would be best if we can keep using it. In the error above, it looks like the reason you're getting an error saying that the autoprefixer plugin requires postcss 8 (when that's the version we already have installed at the root) is because `@storybook/builder-webpack4` must specify some different version of `postcss`--you can tell by the fact that the error path is under `node_modules/@storybook/builder-webpack4/node_modules/postcss` rather than just `node_modules/postcss`. 
![image](https://user-images.githubusercontent.com/5864305/124228131-939c4480-dac0-11eb-9378-8114e4de45eb.png)

So the next thing to look at is whether there's a newer version of `@storybook/builder-webpack4` (and/or `@storybook/addon-info` which apparently pulled it in, based on the stack), and see if that specifies an updated version of `autoprefixer`. Or if all else fails we can add a yarn resolution, or have multiple versions specified and add a syncpack policy exception. (It's also weird that latest storybook would depend on something which presumably uses webpack 4 to begin with...)

Reply:
Yeah i agree about sticking to `autoprefixer` 10. Looking at the yarn.lock file, `@storybook/builder-webpack4` has a dependency on `postcss "^7.0.35"`. This isn't the case if we started using `storybook/builder-webpack5` which was introduced in storybook 6.2 but that's still considered [experimental ](https://gist.github.com/shilman/8856ea1786dcd247139b47b270912324#upgrade)so I didn't want to upgrade as that may introduce more issues. One of the issues is definitely us still using `@storybook/addon-info` which is deprecated as of storybook 6.0. Just to test things out, I removed `storybook/addon-info` and the react and converged packages started up just fine without it and the error involving it disappeared. However, trying to start storybook on `react-components` yielded a similar error (minus the addon-info error) but this one is seemingly rooted in the `@storybook/builder-webpack4` module.

![image](https://user-images.githubusercontent.com/8649804/124301465-bc4e2980-db14-11eb-834c-2999d1552855.png)

Reply:
I'd really like to get off anything using webpack 4 if possible, because overall that's likely to introduce *more* problems since the rest of our repo is on webpack 5.

How are we using addon-info? Can we just remove it?

Reply:
If you search for `addon-info` in the repo I'm only seeing a few references, none of which appear to relate to converged. And it looks like the only thing it's being used for is showing the source code of the stories in v8, which IMO is not critical. So I'd say just remove it for now, and if we need a way to see story source later we can figure that out separately.

Reply:
Ok I'll remove `addon-info` and try to incorporate `@storybook/builder-webpack5`

Reply:
Both tasks above have been completed